install_config/install/quick_install.adoc

```diff
@@ -18,77 +18,215 @@ ifdef::openshift-enterprise[]
+the link:prerequisites.html[Prerequisites] topic, you can continue to running an
+link:#running-an-interactive-installation[interactive] or
+link:#running-an-unattended-installation[unattended] installation.
 
-Before installing OpenShift, you must first link:prerequisites.html[satisfy the
-prerequisites] on your hosts, which includes verifying system and environment
-requirements and properly installing and configuring Docker. After following the
-instructions in the link:prerequisites.html[Prerequisites] topic, you can
-continue to link:#installing-openshift[installing OpenShift].
+[[running-an-interactive-installation]]
 
-[[installing-openshift]]
-== Installing OpenShift Enterprise
+== Running an Interactive Installation
 
-The installation utility is provided at https://install.openshift.com. Visit
-that page for the latest information and to download the portable version if
-necessary.
+[NOTE]
+====
+Ensure you have read through the link:#quick-install-prereqs[Prerequisites]
+before proceeding.
+====
 
-There are two methods for using the installation utility.
+You can start the interactive installation by running:
 
-*Method 1: Running the Installation Utility From the Internet*
+----
+$ atomic-openshift-installer install
+----
 
-. Run the installation utility directly from the Internet using the following
-command on a host that has SSH access to your intended master and node hosts:
-+
+Then follow the on-screen instructions to install a new OpenShift Enterprise
+cluster.
+
+After it has finished, ensure that you back up the
+*_~/.config/openshift/installer.cfg.yml_*
+link:#defining-an-installation-configuration-file[installation configuration
+file] that is created, as it is required if you later want to re-run the
+installation, add hosts to the cluster, or
+link:../../install_config/upgrades.html[upgrade your cluster]. Then, see
+link:#quick-install-whats-next[What's Next] for the next steps on configuring
+your OpenShift cluster.
+
+[[defining-an-installation-configuration-file]]
+
+== Defining an Installation Configuration File
+
+The installation utility can use a predefined installation configuration file,
+which contains information about your installation, host, and cluster
+configurations. If you run an link:#installing-openshift[interactive
+installation], an installation configuration file based on your answers is
+created for you in *_~/.config/openshift/installer.cfg.yml_* if you are instructed
+to exit the installation to manually modify your configuration, or when the
+installation completes. You can also create the configuration file manually from
+scratch to perform an link:#running-an-unattended-installation[unattended
+installation].
+
+.Installation Configuration File Specification
+====
+[source,yaml]
 ----
-$ sh <(curl -s https://install.openshift.com/ose/)
+variant: openshift-enterprise <1>
+variant_version: 3.1 <2>
+ansible_ssh_user: root <3>
```

Comment:
Need to add a line between ansible_ssh_user and hosts:

ansible_log_path: /tmp/ansible.log
Description: Defines where the Ansible logs are stored. By default, this is in *_/tmp/ansible.log_*.